css/main.css

```diff
@@ -325,3 +325,89 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+.contact .container {
+  padding: 5rem 4rem 7rem 4rem;
+  display: flex;
+}
+
+.contact .contact-text {
+  position: static;
+  width: 455px;
+  height: 256px;
+  left: 0;
+  top: 0;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #262626;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 24px 10px 24px 0;
+}
+
+.contact .container form {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0;
+  position: static;
+  width: 684px;
+  height: 330px;
+  left: 479px;
+  top: 130px;
+  margin-top: 24px;
+}
+
+.contact .container form .name-group {
+  display: flex;
+  align-self: stretch;
+  margin-bottom: 20px;
+}
+
+.contact .container form .name-group input {
+  flex-grow: 1;
+  border: 0;
+  border-bottom: solid 3px #dbd8d7;
+  margin-right: 20px;
+  padding: 10px;
+  font-size: 15px;
+  color: #979493;
+  line-height: 24px;
+  font-weight: 400;
+}
+
+.contact .container form .email {
+  border: 0;
+  border-bottom: solid 3px #dbd8d7;
+  padding: 10px;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  color: #979493;
+  margin-bottom: 20px;
+  width: 100%;
+}
+
+.contact .container form textarea {
+  border: 0;
+  background: #fbf8f7;
+  width: 100%;
+  padding: 15px 10px 15px 10px;
+  margin-bottom: 20px;
+}
+
+.contact .container form .btn {
+  padding: 15px;
+  color: #fff;
+  font-size: 17px;
+  font-weight: 700;
+  line-height: 24px;
+  background: #f55800;
+  font-family: "Roboto", sans-serif;
+  border: 0;
+}
```

Comment:
- [ ] It would be better if we add a descriptive README file for our project :+1: Having a professional repository includes having a descriptive README, where we document our project, provide deployed version of our app, instructions about how to clone, and run the app locally, technologies used for building the app, title, short description, etc